### ISSUE TITLE
Addition of New Tutorials and Projects Featuring the Latest Tech Stack

### ISSUE THREAD

**Issue by Polqt** [tuvtran/project-based-learning#620]
### Description

I am proposing the addition of new tutorials and projects that utilize the latest technology stack. This update aims to provide guidance and hands-on experience with current tech trends and tools.

### Why

This change is important to stay up-to-date with the rapidly evolving tech landscape. By incorporating new tutorials and projects, users can learn and apply cutting-edge technologies, which will enhance their skills and make them more competitive in the job market.

(1) Provides updated learning resources for users.
(2) Encourages exploration and experimentation with new tools.
(3) Helps users stay relevant in the tech industry.

### Possible Implementation & Open Questions
(1) Research and compile a list of new technologies and tools that are currently popular or emerging.
(2) Create or curate tutorials and projects that demonstrate practical applications of these technologies.
(3) Organize the content into categories (e.g., web development, data science, AI) for easy navigation.

### Is this something you're interested in working on?

- [✓] YES
- [ ] NO

**Comment by AbhayPundora** [comment]
And what is it going to be? (like a website?)

**Comment by Polqt** [comment]
As a reply to Abhay's comment, you could write:

---

Yes, the goal is to create a web-based platform where users can easily access and engage with these tutorials and projects. The platform would be structured to provide an interactive learning experience, with content organized into categories like web development, data science, and AI. This way, users can navigate based on their interests or skill goals and dive into the latest technologies through hands-on projects. 

Does that approach align with what you had in mind?

**Comment by Polqt** [comment]
Yes, the goal is to create a web-based platform where users can easily access and engage with these tutorials and projects. The platform would be structured to provide an interactive learning experience, with content organized into categories like web development, data science, and AI. This way, users can navigate based on their interests or skill goals and dive into the latest technologies through hands-on projects. 

Does that approach align with what you had in mind?

**Comment by AbhayPundora** [comment]
sorry for late response, yes that's exactly what i'm thinking

**Comment by AbhayPundora** [comment]
so what is the next step ?
